From a30ad013082882bf5ebec073040379f762e10d7f Mon Sep 17 00:00:00 2001
From: Steve Loughran <stevel@apache.org>
Date: Mon, 29 Jan 2018 09:48:14 -0800
Subject: [PATCH 2793/2863] HADOOP-15186. Allow Azure Data Lake SDK dependency
 version to be set on the command line.
 Contributed by Vishwajeet Dusane.

(cherry picked from commit 95a96b13e2a54e01ea6c6933045d912998477da3)
(cherry picked from commit 7fd287b4af5a191f18ea92850b7d904e4b4fb693)

Change-Id: Ic91d62e708bd70115ce3b983b7c2d5b9764e7ac5
(cherry picked from commit cceb62fba4d11cabad00c3e2b8e5a9696dc1e177)
---
 hadoop-tools/hadoop-azure-datalake/pom.xml |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hadoop-tools/hadoop-azure-datalake/pom.xml b/hadoop-tools/hadoop-azure-datalake/pom.xml
index 0249eb8..b93f329 100644
--- a/hadoop-tools/hadoop-azure-datalake/pom.xml
+++ b/hadoop-tools/hadoop-azure-datalake/pom.xml
@@ -34,6 +34,7 @@
     <minimalJsonVersion>0.9.1</minimalJsonVersion>
     <file.encoding>UTF-8</file.encoding>
     <downloadSources>true</downloadSources>
+    <azure.data.lake.store.sdk.version>2.2.5</azure.data.lake.store.sdk.version>
   </properties>
   <build>
     <plugins>
@@ -110,7 +111,7 @@
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>azure-data-lake-store-sdk</artifactId>
-      <version>2.2.5</version>
+      <version>${azure.data.lake.store.sdk.version}</version>
     </dependency>
     <!--  ENDS HERE-->
     <dependency>
-- 
1.7.9.5

